web/profiles/custom/yalesites_profile/modules/custom/ys_templated_content/src/Modifiers/TemplateModifier.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Drupal\ys_templated_content\Modifiers;
+
+use Drupal\Component\Uuid\UuidInterface;
+use Drupal\path_alias\AliasRepositoryInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Modifies a content import for a unique insertion.
+ */
+class TemplateModifier {
+
+  const PLACEHOLDER = 'public://templated-content-images/placeholder.png';
```

Comment:
I like your approach. Let's get the design/UX team to provide a real Yale branded image.

Reply:
Good idea.  Ya I worried a lot about the idea that an image might not exist on the page, but we'd still want it to load something so that they knew they would need to change it.